src/System.Net.Primitives/tests/UnitTests/CookieContainerTest.cs

```diff
@@ -352,6 +352,28 @@ public void SetCookies_Success(Uri uri, string cookieHeader, Cookie[] expected)
             VerifyGetCookies(cc, uri, expected);
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Requires fix shipping in .NET 4.7.2")]
```

Comment:
There is no fix in .NET 4.7.2 for this.  So, this should probably say "UTC timezone not supported"

Reply:
Updated